app/models/manageiq/providers/cloud_manager/orchestration_stack.rb

```diff
@@ -16,4 +19,13 @@ def self.create_stack(orchestration_manager, stack_name, template, options = {})
                  :cloud_tenant           => tenant,
                  :orchestration_template => template)
   end
+
+  def self.orchestration_stack_class_factory(orchestration_manager, template)
```

Comment:
I don't like the special handling for `OrchestrationTemplateVnfd`. This can serve as a temporary solution. When tacker becomes a separated provider, we may change the `Vnf` class to `OrchestrationStack` under the new provider. We may create alias `Vnf` for the `OrchestrationStack` class.